packages/@aws-cdk/aws-cloudwatch/test/test.metrics.ts

```diff
@@ -97,6 +97,17 @@ export = {
       });
     }, `Dimension value must be at least 1 and no more than 255 characters; received ${invalidDimensionValue}`);
 
+    test.throws(() => {
```

Comment:
I would create a separate test for this - will be easier to migrate on V2, where the deprecated properties are removed.